CHANGELOG.md

```diff
@@ -5,11 +5,16 @@
 ### Added
 - Add `vendor[1,2,3]-prompt` as options to ConnectionConfig-CT in `tcd_latest.xsd`.
```

Comment:
Released, [PR](https://github.com/tableau/connector-plugin-sdk/pull/562)

Reply:
Will remove this line